app/scripts/views/base.js

```diff
@@ -86,6 +86,15 @@ define(function (require, exports, module) {
     var translated = this.translateError(err);
 
     var $error = this.$('.error');
+
+    if (err.errno && err.errno === 1005) {
```

Comment:
Should this reference the working errno constant?

Reply:
good idea!

Reply:
You can do `AuthErrors.is(err, 'WORKING')`

Reply:
You can do AuthErrors.is(err, 'WORKING')